9.2/root/usr/share/container-scripts/postgresql/common.sh

```diff
@@ -114,6 +114,19 @@ function generate_postgresql_recovery_config() {
       > "${POSTGRESQL_RECOVERY_FILE}"
 }
 
+# this function iterates over hooks.d and seek for folders
+# which contains post-init executables
+function execture_post_init_scripts() {
+  for i in ${CONTAINER_SCRIPTS_PATH}/hooks.d/*
```

Comment:
Then we could have `pre-init.d`, `post-init.d`, and `hooks.d` as an alias for `post-init.d`, for example.  But I vote fore renaming, yes.

The default value is in Dockerfile.